common.sh

```diff
@@ -126,6 +126,12 @@ if [[ "$OPENSHIFT_RELEASE_TYPE" == "ga" ]]; then
     export OPENSHIFT_RELEASE_STREAM=${OPENSHIFT_VERSION%.*}
 fi
 
+if [ "${FIPS_MODE:-false}" = "true" ]; then
+    if ! [ "${FIPS_VALIDATE:-false}" = "true" ]; then
+        export OPENSHIFT_INSTALL_SKIP_HOSTCRYPT_VALIDATION="true"
```

Comment:
Where's getting consumed this var? Wasn't able to find any reference in openshift install

Reply:
It doesn't exist yet, mention in https://issues.redhat.com/browse/OCPBUGS-15845

Reply:
Thanks, I've missed that.

/approve